src/main/java/mat/server/service/FhirValidationReportService.java

```diff
@@ -155,7 +156,9 @@ private String generateValidationReport(ConversionResultDto conversionResultDto,
 
                 //Measure FHIR validation errors
                 List<FhirValidationResult> measureFhirValidationErrors = new ArrayList<>();
```

Comment:
Consider lazy instantiating the collections holding the reporting values throughout. (GH won't let me comment on code outside of the changed areas.)

Now I'm wondering, does lazy init do anything after this gets run through GWT?

Reply:
@jkotanchik-SB - this is server-side code. Not gwt

Reply:
In that case, lazy init collections, when applied throughout, would reduce our footprint, though not by a lot.